types/jasmine/index.d.ts

```diff
@@ -676,6 +676,12 @@ declare namespace jasmine {
     }
 
     interface AsyncMatchers<T, U> {
+        /**
+         * Expect a promise to be pending, ie. the promise is neither resolved nor rejected.
```

Comment:
uber nit: i.e.

Reply:
good catch! this typo exists verbatim in the source library, too. this is the first time I've contributed to DefinitelyTyped, do you know if there is a convention on fixing comments that are copy pasted from the library?

Reply:
Not that I know of, but should be simple to fix it in the lib as well?